imap/httpd.h

```diff
@@ -344,7 +344,8 @@ struct http_connection {
 
     void *tls_ctx;                      /* TLS context */
     void *sess_ctx;                     /* HTTP/2+ session context */
-    void **ws_ctx;                      /* WebSocket context (HTTP/1.1 only) */
+    const void *ws_ctx;                 /* Reference to WebSocket context
+                                           (HTTP/1.1 only) */
```

Comment:
I think this doesn't need to be `void` anymore either, since it's not used polymorphically, so it could just be a `const ws_context *`... though everything else here is `void *`, and maybe it's easier to just keep thinking about `void *` once you are already.

Reply:
Its easier to keep it void* in case we're compiled without wslay.